airflow/www/utils.py

```diff
@@ -238,6 +247,8 @@ def generate_pages(current_page, num_of_pages, search=None, status=None, tags=No
     :param status: 'all', 'active', or 'paused'
     :param tags: array of strings of the current filtered tags
     :param window: the number of pages to be shown in the paging component (7 default)
+    :param sorting_key: the sorting key selected for the dags
+    :param sorting_direction: direction of sorting, 'asc' or 'desc'
```

Comment:
What does providing `None` to either param mean? Can this be described in the docstring? Can only one of those be None and the other not?

Reply:
Having it as None means that sorting is not applicable or not provided. I will mention that in the docstring as well.

Good point. It makes sense to have all or none behaving in a similar way as they are dependent variables. That is why I am using them in an "and" way, for ex: https://github.com/apache/airflow/pull/29756/files#diff-ba09c7727b94aa05a7557d151820479d3f53d71e9a4c601db8a3de3f7f2caf76R42